### ISSUE TITLE
Get rid of `Δ` attribute in `tuple`

### ISSUE THREAD

**Issue by maxonfjvipon** [objectionary/eo#2555]
In order to resolve #2437 and leave `Δ` only in `bytes` we need to get rid of it on `tuple`. 
Implementation of the `tuple` should be recursive and similar to implementation of `tuple` in LISP.

**Comment by maxonfjvipon** [comment]
New recursive realisation of `tuple` causes recursion between object `tuple.length` and `int.plus`:
```
[head tail] > tuple
  [] > length
    ^.head.length.plus 1 > @ # int.plus is called here
------
[] > int
  [x...] > plus /int # x is tuple here
```
In order to add up all the numbers of `x` we need to get `x.length`. And here we get recursion `tuple.length -> int.plus -> tuple.length -> int.plus...`.
It can be resolved only if we get rid of varargs in `int.plus` object (and in all primitives in `eo-runtime`)

**Comment by 0pdd** [comment]
@maxonfjvipon 5 puzzles [#2561](https://github.com/objectionary/eo/issues/2561), [#2562](https://github.com/objectionary/eo/issues/2562), [#2563](https://github.com/objectionary/eo/issues/2563), [#2564](https://github.com/objectionary/eo/issues/2564), [#2565](https://github.com/objectionary/eo/issues/2565) are still not solved.

**Comment by 0pdd** [comment]
@maxonfjvipon 4 puzzles [#2561](https://github.com/objectionary/eo/issues/2561), [#2562](https://github.com/objectionary/eo/issues/2562), [#2563](https://github.com/objectionary/eo/issues/2563), [#2564](https://github.com/objectionary/eo/issues/2564) are still not solved; solved: [#2565](https://github.com/objectionary/eo/issues/2565).

**Comment by 0pdd** [comment]
@maxonfjvipon 3 puzzles [#2561](https://github.com/objectionary/eo/issues/2561), [#2562](https://github.com/objectionary/eo/issues/2562), [#2563](https://github.com/objectionary/eo/issues/2563) are still not solved; solved: [#2564](https://github.com/objectionary/eo/issues/2564), [#2565](https://github.com/objectionary/eo/issues/2565).

**Comment by 0pdd** [comment]
@maxonfjvipon 2 puzzles [#2561](https://github.com/objectionary/eo/issues/2561), [#2563](https://github.com/objectionary/eo/issues/2563) are still not solved; solved: [#2562](https://github.com/objectionary/eo/issues/2562), [#2564](https://github.com/objectionary/eo/issues/2564), [#2565](https://github.com/objectionary/eo/issues/2565).

**Comment by 0pdd** [comment]
@maxonfjvipon all 5 puzzles are solved here: [#2561](https://github.com/objectionary/eo/issues/2561), [#2562](https://github.com/objectionary/eo/issues/2562), [#2563](https://github.com/objectionary/eo/issues/2563), [#2564](https://github.com/objectionary/eo/issues/2564), [#2565](https://github.com/objectionary/eo/issues/2565).
